modules/ROOT/pages/about-mule-message.adoc

```diff
@@ -119,13 +120,16 @@ For example, when using the `attributes` variable to display HTTP response metad
 
 The example is a response from an HTTP Request operation to the web page `+https://jsonplaceholder.typicode.com/users+`.
 
-Notice that each attribute is a key/value pair separated by an equal sign (`=`). You access the value of an attribute through its key. For example, when using a connector or component (such as the Logger) in your flow, you can access attributes of an HTTP response or a file that you are reading with the following syntax:
+[[select_attribute]]
+Notice that each attribute is a key-value pair separated by an equal sign (`=`). When using a connector or component (such as the Logger) in your flow, you can use the `attribute` variable with specific attributes and their inner attributes to select the attribute values that you require, such as the status code or content type. For example, you can select the values of attributes in an HTTP response using the following syntax:
 
-* `attributes.'statusCode'` to select an HTTP status code like `200`.
-* `attributes.headers.'date'` to select `Sun, 20 Jan 2019 18:54:54 GMT` from the header of an HTTP response.
+* `attributes.statusCode` to select an HTTP status code like `200`.
+* `attributes.headers.date` to select `Sun, 20 Jan 2019 18:54:54 GMT` from the header of an HTTP response.
 * `attributes.headers.'content-type'` to select the HTTP content type `application/json`.
++
+Quotes are required around the inner attribute values such as `content-type` but not around inner values `date` because DataWeave applies the rules described in <<valid_identifiers>> to inner values.  To understand why you use `statusCode` instead of `Status Code` or `"Status Code"` to select the status code value, see the <<note_top_level_selectors, note>> in that section.
```

Comment:
The first sentence confused me a bit the first time I read it, could it just be simplified to something like this? : 
"Quotes are required to select the inner attribute `content-type` because it does not follow the Rules of a <<valid_identifiers, valid identifier>>" 
In the second sentence, I'd remove the **or `"Status Code"`**, because it seems we are saying that using quotes there is wrong but it's actually optional, as in any other attribute name.

Reply:
@Cristian-Venticinque Using quotes around "Status Code" does not work there. The value is null if you do that. That's why it needs to be clarified here.

Reply:
tweaked the language a bit.